client/checkout/platform-checkout/email-input-iframe.js

```diff
@@ -297,6 +351,29 @@ export const handlePlatformCheckoutEmailInput = ( field, api ) => {
 		}
 
 		switch ( e.data.action ) {
+			case 'auto_redirect_to_platform_checkout':
+				hasCheckedLoginSession = true;
+				loginSessionIframeWrapper.classList.add(
+					'platform-checkout-login-session-iframe-wrapper'
+				);
+				loginSessionIframe.classList.add( 'open' );
+				wcpayTracks.recordUserEvent(
+					wcpayTracks.events.PLATFORM_CHECKOUT_AUTO_REDIRECT
+				);
+				api.initPlatformCheckout(
+					e.data.userEmail,
```

Comment:
Do we still have `e.data.userEmail` for `auto_redirect_to_platform_checkout`?

Reply:
No actually. I can send `""` there as well.